aircraft.py

```diff
@@ -939,8 +956,6 @@ def setup(self, Ncruise, objective, airplane, Nmission = 1):
 
                 cruise['hft'] == MinCruiseAlt,
 
-                cruise['M'] == 0.8,
-
```

Comment:
@1ozturkbe @pgkirsch interesting to note in the solution the second flight segment M is now 0.83 since the engine model gives free thrust with no Mach number penalty on TSFC